src/Files.App/Helpers/LocationHelpers.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright(c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using System.Text.Json;
+using Windows.Devices.Geolocation;
+using Windows.Services.Maps;
+using Windows.Storage;
+
+namespace Files.App.Helpers
+{
+	public static class LocationHelpers
+	{
+		public static async Task<string> GetAddressFromCoordinatesAsync(double? Lat, double? Lon)
+		{
+			if (!Lat.HasValue || !Lon.HasValue)
+				return null;
+
+			try
+			{
+				StorageFile file = await StorageFile.GetFileFromApplicationUriAsync(new Uri(@"ms-appx:///Resources/BingMapsKey.txt"));
```

Comment:
Should we cache the result instead of reloading the key for every file?

Reply:
Updated.